Shared/Samples/Find route around barriers/FindRouteAroundBarriersView.Model.swift

```diff
@@ -0,0 +1,207 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension FindRouteAroundBarriersView {
+    /// The view model for the sample.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// A map with a topographic basemap cantered on San Diego, CA, USA.
+        let map = {
+            let map = Map(basemapStyle: .arcGISTopographic)
+            map.initialViewpoint = Viewpoint(
+                center: Point(x: -13_042_200, y: 3_857_900, spatialReference: .webMercator),
+                scale: 1e5)
+            return map
+        }()
+        
+        /// The graphics overlays for the sample.
+        let graphicsOverlays: [GraphicsOverlay]
+        
+        /// The graphics overlay for the stop graphics.
+        private let stopGraphicsOverlay = GraphicsOverlay()
+        
+        /// The graphics overlay for the barrier graphics.
+        private let barrierGraphicsOverlay = GraphicsOverlay()
+        
+        /// The graphics overlay for the route graphics.
+        private let routeGraphicsOverlay = GraphicsOverlay()
+        
+        /// The blue marker symbol for the stop graphics.
+        private let stopSymbol = {
+            let markerImage = UIImage(named: "BlueMarker")!
+            let markerSymbol = PictureMarkerSymbol(image: markerImage)
+            markerSymbol.offsetY = markerImage.size.height / 2
+            return markerSymbol
+        }()
+        
+        /// The graphic for the route.
+        private let routeGraphic = Graphic(
+            symbol: SimpleLineSymbol(style: .solid, color: .yellow, width: 5)
+        )
+        
+        /// The graphic for the direction route.
+        let directionGraphic = Graphic(
+            symbol: SimpleLineSymbol(style: .dashDot, color: .orange, width: 5)
+        )
+        
+        /// The route task for routing.
+        let routeTask = RouteTask(url: .sanDiegoNetworkAnalysis)
+        
+        /// The route parameters for routing with the route task.
+        var routeParameters: RouteParameters?
+        
+        /// The resulting route from a routing operation with the route task.
+        private(set) var route: Route?
+        
+        /// The text with the time and distance of the current route.
+        @Published private(set) var routeInfoText = ""
+        
+        /// The count of stops currently on the map.
+        @Published private(set) var stopsCount: Int = 0
```

Comment:
Nitpick: No need to specify the type
```suggestion
        @Published private(set) var stopsCount = 0
```